src/FwdState.cc

```diff
@@ -1068,6 +1045,16 @@ FwdState::usePinned()
         return;
     }
 
+    if (connManager->pinning.peerAccessDenied) {
+        // The peer selection policy denies access to pinned connection.
```

Comment:
I would remove these comment lines. They only retell what the code already clearly says.

Reply:
resolved with commit 22aad31